spec/input_objects/account/terms_of_use_input_spec.rb

```diff
@@ -0,0 +1,56 @@
+require "rails_helper"
+
+describe Account::TermsOfUseInput do
+  include ActiveSupport::Testing::TimeHelpers
+
+  subject(:terms_of_use_input) { described_class.new(user:) }
+
+  let(:user) { create(:user) }
+
+  describe "validations" do
+    it "is valid with an agreed attribute" do
+      terms_of_use_input.agreed = true
+      expect(terms_of_use_input).to be_valid
+    end
+
+    it "is invalid without an agreed attribute" do
```

Comment:
I don't think this test name quite fits what it does - 'without an agreed attribute' makes me expect `terms_of_use_input.agreed` to be `nil` or `undefined`.

```suggestion
    it "is invalid when the user has not agreed" do
```

```suggestion
    it "is invalid when the agreed attribute is false" do
```

(This is nitpicky so don't let it block you if you don't agree)

Reply:
Good suggestion, I've pushed up a change for this.

Thanks!